R/combine_omicsData.R

```diff
@@ -74,7 +80,8 @@ combine_lipidData <- function(obj_1, obj_2, retain_groups = FALSE, retain_filter
   }
 
   # Check that it is among supported objects
-  if (!(class(obj_1) %in% c("lipidData"))) stop("Currently only support lipidData")
+  if (!(class(obj_1) %in% c("lipidData", "metabData"))) 
```

Comment:
small thing can we change this to `!inherits(obj_1, c("lipidData", "metabData"))`, it will throw an uninformative error if they pass like a tibble or something